### DISCUSSION TITLE
Multiple services in one LXC

### DISCUSSION THREAD

**Discussion by andoni-galardi** [tteck/Proxmox#3883] [Ideas]
Hi,

I would like to create a single LXC with multiple services (caddy, guacamole, homer, adguard, wireguard, etc.).

I used to create a LXC per service with the scripts, but I would prefer to have a single LXC for all to save resources.

Do you think it could be possible a script that let you select as much services or products as you want to install them?

Maybe the most dificult part is to know if there are incompatiblilities and to decide the minimum resources.

Regards

**Comment by tteck** [comment]
It's not going to happen.

**Comment by andoni-galardi** [comment]
Ok :(, thanks for your answer
